src/TLuaInterpreter.cpp

```diff
@@ -7129,13 +7130,13 @@ int TLuaInterpreter::setAreaName(lua_State* L)
 
     bool result = host.mpMap->mpRoomDB->setAreaName(id, newName);
     if (result) {
-        // Update mapper Area names widget, using method designed for it...!
         if (host.mpMap->mpMapper) {
             host.mpMap->mpMapper->updateAreaComboBox();
             if (isCurrentAreaRenamed) {
                 host.mpMap->mpMapper->showArea->setCurrentText(newName);
             }
         }
+        updateMap(L);
```

Comment:
I think this should be *inside* the preceding `}` as it doesn't make sense to call `updateMap()` if there is no map displayed to update...

Reply:
OK, done.